assets/bem/styles.scss

```diff
@@ -2,7 +2,7 @@
 @import "core";
```

Comment:
hm...why styles is in assets folder? Assets meant to be an icon/picture/other resources folder but not css/scss/js

Reply:
I did that structure at my last work, and I think that it is ok, can you please send some information about best practice in this question?